projects/_template/index.js

```diff
@@ -24,7 +24,7 @@
   module.exports = {
     name: 'Template Project', // project name
     token: null,              // null, or token symbol if project has a custom token
-    category: 'assets',       // allowed values as shown on DefiPulse: 'Derivatives', 'DEXes', 'Lending', 'Payments', 'Assets'
+    category: 'Assets',       // allowed values as shown on DefiPulse: 'Derivatives', 'DEXes', 'Lending', 'Payments', 'Assets'
```

Comment:
just a housekeeping fix, noticed that this fails due to case-sensitivity for category type

Reply:
this is probably why "assets" was way more commonly an issue than the others....